### ISSUE TITLE
Add builds for Linux and Macos

### ISSUE THREAD

**Issue by denisgolius** [vv9k/dockeye#35]
#### Describe your feature request
It will be nice feature if you will add cross-platform CI builds which will be upload binary files to github releases because sometimes people don't need to install rust on their machines.
Thanks!

**Comment by vv9k** [comment]
Hey, thanks for reporting the issue. I'm pretty sure it's described in the readme and already implemented :)

https://github.com/vv9k/dockeye/blob/4f0a86b9a069ab0b7ac6bed82b7f16fbc48eec36/.github/workflows/release.yml#L129-L148

https://github.com/vv9k/dockeye/blob/master/README.md#prebuilt-packages

**Comment by vv9k** [comment]
Here is the first release with Linux and macOS build artifacts: https://github.com/vv9k/dockeye/releases/tag/0.1.0

**Comment by denisgolius** [comment]
I don't see Mac x86_64 architecture builds, only aarch64.

**Comment by vv9k** [comment]
I added the x86_64-apple-darwin target in 8014a26ed6ee9ac5f0d0eb295415ec739ebe0e07 . Next release should include the artifacts :) Let's leave this issue open until the next release.

**Comment by SebDanielsson** [comment]
Any plans for adding Linux aarch64/arm64 bin to the release?😃

**Comment by denisgolius** [comment]
> Any plans for adding Linux aarch64/arm64 bin to the release?smiley

@vv9k sound like you need to add "help neded" label 🙈

**Comment by vv9k** [comment]
Agreed :) All help at this point will be appreciated and Linux arm is indeed not on my priority list.

Necessary changes would have to be made here to add a new target: https://github.com/vv9k/dockeye/blob/fd85696e5980bad4f1f061e82068658c36eb25a7/.github/workflows/release.yml#L69-L73

**Comment by vv9k** [comment]
Looks like the binaries for x86_64-apple-darwin are there! :) Perhaps I'll rename this issue track adding the linux arm binary.
